vipgo-helper.php

```diff
@@ -1,4 +1,12 @@
 <?php
+/**
+ * VIP Go Helper
+ *
+ * @package VIP_Maintenance_Mode
+ *
+ * @phpcs:disable PHPCompatibility.FunctionDeclarations.NewReturnTypeDeclarations.boolFound -- return type declaration invalid on php<7, VIP Go is 7+
```

Comment:
The `composer.json` claims this plugin only requires PHP 5.6.

The readme.txt doesn't have a "Requires PHP:" header.

Either the requirement in the `composer.json` should change to be a version of PHP that supports return type declarations (and then bump the `testVersion` value for PHPCS to match), or the return type declaration removed. Can't have it both ways :-)

Reply:
In practice, `vipgo-helper.php` doesn't need to meet the minimum of the rest of the plugin. It is not loaded by the plugin itself, but by the VIP Go platform.

Though I guess it doesn't hurt to remove the return type declaration.

Reply:
Let's find a suitably comprehensive boilerplate text to put into the top of `vipgo-helper.php` file-level DocBlock then, to explain what the file is, and how it's used, with links to any public docs we have (or mu-plugin readme / reference for it).

Reply:
that's entirely fair.